examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +179,83 @@
 plt.tight_layout()
 
 plt.show()
+
+# %%
+# From the plot above we observe that scaling the features before reducing the
+# dimensionality results in components with the same order of magnitude. In this
+# case it also improves the separability of the clases. Indeed, in the next
+# section we confirm that a better separability has a good repercussion on the
+# overall model's performance.
+#
+# Effect of rescaling on model's performance
+# ==========================================
+#
+# First we show how the optimal regularization of a
+# class:`~sklearn.linear_model.LogisticRegressionCV` depends on the scaling or
+# non-scaling of the data:
+
+import numpy as np
+from sklearn.pipeline import make_pipeline
+from sklearn.linear_model import LogisticRegressionCV
+
+Cs = np.logspace(-5, 5, 20)
+
+unscaled_clf = make_pipeline(pca, LogisticRegressionCV(Cs=Cs))
+unscaled_clf.fit(X_train, y_train)
+y_pred = unscaled_clf.predict(X_test)
+
+scaled_clf = make_pipeline(scaler, pca, LogisticRegressionCV(Cs=Cs))
+scaled_clf.fit(X_train, y_train)
+y_pred_scaled = scaled_clf.predict(X_test)
+
+print("Optimal C for the unscaled PCA")
+print(unscaled_clf[-1].C_)
```

Comment:
That is because we are using a grid and the same value is getting selected in each fold. Would you suggest to use a random search, smaller grid lattice or just show one value?

Reply:
Just show one single value.